plugins/frontend/backstage-plugin-github-pull-requests/src/components/PullRequestsStatsCard/PullRequestsStatsCard.test.tsx

```diff
@@ -50,7 +50,7 @@ describe('PullRequestsCard', () => {
   beforeEach(() => {

```

Comment:
I see previous lines about config, ApiRegistry are copied in each test file ... it was done like that before. what do you think about a common file ?